src/main/java/org/alliancegenome/curation_api/constants/Gff3Constants.java

```diff
@@ -0,0 +1,15 @@
+package org.alliancegenome.curation_api.constants;
+
+import java.util.List;
+
+import lombok.experimental.UtilityClass;
+
+@UtilityClass
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.design.HideUtilityClassConstructorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Utility classes should not have a public or default constructor.